scripts/compare_benchmarks.py

```diff
@@ -12,10 +12,20 @@
 min_runtime_ns = 59 * 1000 * 1000 * 1000
 
 def format_diff(diff):
-    if diff >= 0:
-        return colored('+' + "{0:.0%}".format(diff), 'green')
+    select_color = lambda value, color: color if abs(value) > 0.05 else 'white'
```

Comment:
Technically, this is still biased. The white range should be (1/1,05, 1,05). The difference is 0.0023809524, so...